docs/pages/access-controls/sso/github-sso.mdx

```diff
@@ -67,8 +68,8 @@ spec:
         - access
 ```
 
-</ScopedBlock>
-<ScopedBlock scope="enterprise">
+</TabItem>
+<TabItem label="Enterprise" scope="enterprise">
```

Comment:
Wouldn't this apply to cloud as well as you mentioned on the original PR?

Reply:
Now that I know it does, I'll adjust:

```suggestion
<TabItem label="Enterprise / Cloud" scope={["enterprise", "cloud"]}>
```